tests/robot-tests/tests/general_public/release_page_absence.robot

```diff
@@ -110,7 +110,7 @@ Validate headlines -- Summary tab content
 
 Validate Key Statistics data block -- Charts tab
     [Tags]  HappyPath
-    user scrolls to element  css:#releaseHeadlines-chart 
+    user scrolls down  400
```

Comment:
I wouldn't think this would be better than scrolling to a specific element. Why change this?

Reply:
For some reason when Selenium attempted to scroll to that element it caused the below error on the Azure pipeline: 
```
ElementNotInteractableException: Message: element not interactable: [object HTMLElement] has no size and location
```
Using the `user scrolls down` keyword seems to prevent this

Reply:
I've investigated this. It was failing on the `user scrolls to element` because `releaseHeadlines-chart` element wasn't visible because the Table tab was currently selected. So I've updated this to `user scrolls to element  css:#releaseHeadlines-charts-tab`